src/systemctl/systemctl-show.c

```diff
@@ -433,6 +434,18 @@ static void print_status_info(
                                FORMAT_TIMESTAMP_STYLE(until_timestamp, arg_timestamp_style),
                                FORMAT_TIMESTAMP_RELATIVE(until_timestamp));
                 }
+
+                if (!streq_ptr(i->type, "target") &&
```

Comment:
so far we just did `!endswith(i->id, ".target")` for checks like this. Should be good enough for this too, and simpler